lib/block.ml

```diff
@@ -288,6 +289,15 @@ let unset_variables t =
   in
   List.map f (get_prefixed_labels t "unset-")
 
+let required_packages t =
+  let f = function
+    | `Eq, "" ->
+      Fmt.failwith "invalid `require-package` label value: requires a value"
+    | `Eq, pkg -> pkg
+    | _ -> Fmt.failwith "invalid `env` label value"
```

Comment:
Shouldn't it be `` invalid `require-package` label value ``?

Reply:
Nice catch